File: 数据库/NoSQL/4_7/Java驱动程序/MongoDBConnect.java
package cn.itcast.mongodb;
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
public class MongoDBConnect {
    private static final String URI = "mongodb://itcastAdmin:123456@" +
            "192.168.121.134:27016/?authSource=admin";
    private static MongoClient mongoClient;
    public static MongoClient getConnection() {
        if (mongoClient == null) {
            mongoClient = MongoClients.create(URI);
        }
        return mongoClient;
    }
    public static void closeConnection() {
        if (mongoClient != null) {
            mongoClient.close();
            mongoClient = null;
        }
    }
}

File: 数据库/NoSQL/MongoBD安全访问控制和java、python的驱动示例/Python驱动程序/mongodb_connect.py
from pymongo import MongoClient
class MongoDBConnect:
    default_uri = ("mongodb://itcastAdmin:123456@"
                   "192.168.121.134:27016/?authSource=admin")
    @staticmethod
    def getConnect(uri=default_uri):
        client = MongoClient(uri)
        return client
